src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
+  border-bottom 1px solid $color-grey
+  .dollar-now-value-label
+    font-size: 12px
+  .dollar-now-value-field
```

Comment:
Maybe doesn't make sense we keep the dollert prefix. So, i think you can keep it as it is and I will remove all the other dollert prefix used.